app/src/androidTest/java/org/simple/clinic/patient/PatientRepositoryAndroidTest.kt

```diff
@@ -1894,4 +1894,21 @@ class PatientRepositoryAndroidTest {
     savePatientWithBp(facilityUuid = facilityUuid, patientStatus = INACTIVE)
     verifyRecentPatientOrder(patient2, patient1, facilityUuid = facilityUuid)
   }
+
+  @Test
+  fun verify_recordedAt_is_being_used_for_bp_instead_of_updatedAt_for_recent_patients() {
```

Comment:
Happy to see that BPs now have a `recordedAt` timestamp.

Reply:
While you are here, you might as well review some PRs 😛

Reply:
Sure, what PRs do you want to be ~auto-approved~ reviewed?